documentation/source/install.rst

```diff
@@ -0,0 +1,23 @@
+.. _install:
+
+************
+Installation
+************
+
+Using cocotb requires :ref:`installation of pre-requisites<install-pre-requisites>` and
```

Comment:
```suggestion
Using cocotb requires :ref:`installation of prerequisites<install-prerequisites>` and
```

Reply:
Unfortunately, since I an ``.. include::``ing the file where this anchor used to live several times for the different OSes, we cannot really link to it anymore, so I removed it now.
I changed the spelling as suggested.